tests/parser/ReservoirCouplingTests.cpp

```diff
@@ -67,6 +131,43 @@ SLAVES
     BOOST_CHECK_EQUAL(slave.numprocs(), 4);
 }
 
+BOOST_AUTO_TEST_CASE(SLAVES_FAIL_NEGATIVE_NUMPROCS) {
+    std::string deck_string = R"(
+SCHEDULE
+SLAVES
+  'RES-1'  'RC-01_MOD1_PRED'   1*  '../mod1'  -4 /
+  'RES-2'  'RC-01_MOD2_PRED'   1*  '../mod2'  1 /
+/
+
+)";
+
+    assertRaisesInputErrorException(deck_string, /*slave_mode=*/false, /*exception_string=*/ "Problem with keyword SLAVES\nIn <memory string> line 3\nNumber of processors must be positive. Got: -4.");
```

Comment:
Wow. You are really thorough. :muscle: 
I would never have thought about checking the exact error message. If someone changes that they need to reflect the change here, too.

Reply:
Thanks. Yes, the test needs to be updated if the warning message changes. But I guess that goes for all test cases: If the code the test is testing changes the test itself must be also updated ?